server/index.js

```diff
@@ -18,20 +19,65 @@ app.get('/api/buytest', (req, res) => {
       res.send(err);
     } else {
       res.send(docs);
-    }    
+    }
   });
 });
 
-app.get('/api/bpowertest', (req, res) =>{
-  dpower.find((err, docs) => {
+app.get('/api/buytest/one/:id', (req, res) => {
+  const id = JSON.parse(req.params.id);
+  modalBuy.read(id, (err, doc) => {
     if (err) {
-      res.send(err);
+      res.status(500).send(err);
     } else {
-      res.send(docs);
-    }    
-  });
+      res.status(200).send(JSON.stringify(doc))
+    }
+  })
+});
+
+app.post('/api/buytest/:id', (req, res) => {
+  const param = req.body;
+  modalBuy.create(param, (err, success) => {
+    if (err) {
+      res.status(500).send();
+    } else {
+      res.status(201).send('lovin life');
+    }
+  })
+});
+// app.post('/api/bpowertest', (req, res) => {
```

Comment:
You can probably delete this if you aren't using it

Reply:
Deleted!